packages/clay-toolbar/src/index.tsx

```diff
@@ -43,29 +40,24 @@ const ClayToolbar: React.FunctionComponent<IProps> & {
 } = ({
 	children,
 	className,
-	component,
 	inlineBreakpoint,
 	subnav,
 	...otherProps
 }: IProps) => {
-	const classes = classNames(
-		className,
-		'tbar',
-		{
-			'component-tbar': component,
-			'subnav-tbar': !!subnav,
-			'subnav-tbar-disabled': !!(
-				subnav &&
-				typeof subnav !== 'boolean' &&
-				subnav.disabled
-			),
-			[`tbar-inline-${inlineBreakpoint}-down`]: inlineBreakpoint,
-		},
-		subnav &&
-			typeof subnav !== 'boolean' &&
-			subnav.displayType &&
-			`subnav-tbar-${subnav.displayType}`
-	);
+	subnav =
+		typeof subnav === 'boolean'
+			? subnav === true
+				? {}
+				: undefined
+			: subnav;
+
+	const classes = classNames(className, 'component-tbar tbar', {
+		'component-tbar': !subnav,
```

Comment:
Since this class always exists if its not a subnav, I removed the prop and did this